gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-container/src/main/java/io/gravitee/rest/api/standalone/jetty/JettyEmbeddedContainer.java

```diff
@@ -53,8 +53,9 @@
 public final class JettyEmbeddedContainer extends AbstractLifecycleComponent<JettyEmbeddedContainer> implements ApplicationContextAware {
 
     @Autowired
-    private Server server;
+    private JettyServerFactory jettyServerFactory;
 
+    private Server server;
     private ApplicationContext applicationContext;
 
     @Value("${http.api.management.enabled:true}")
```

Comment:
Did you forget this one ?

Reply:
I tried to focus only on classes that make use of `@Value` with sensitive properties susceptible to being held by a secret or beans that are initialized too early. This class wasn't particularly concerned.